apps/freyr/src/pages/index.tsx

```diff
@@ -11,11 +12,10 @@ const IndexPage: FC = () => (
     <SEO title="Home" />
     <div className="flex-row justify-center my-12 max-w-xl">
       <h1 className="text-5xl leading-none font-heading font-medium">
-        Run your Montessori school efficiently
```

Comment:
so we still put the real text as the key even if it's too long?

Reply:
yes, very long keys/msgid are fine. We're editing in .po not .json. Values/msgstr lie under the key, so even the key is too long we could still read the values easily.  And we won't ever need to type the key in manually anyway since lingui generates everything.

.po files are created to be used that way. (see: http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html).